cascade/proxima/__init__.py

```diff
@@ -255,6 +299,7 @@ def get_state(self) -> dict[str, Any]:
         output = {
             'threshold': self.threshold,
             'alpha': self.alpha,
+            'blending_step': int(self.blending_step), # not sure if necessary...
```

Comment:
The `float` is there to convert from a `numpy.float32`, which is not serializable to JSON, to a Python `float`, which is. You shouldn't need to worry about that problem for your field.

Reply:
Actually, since I use `np.clip` to keep the value bounded, the property gets converted to a `numpy.int64` so I do need to convert it for it to be JSON serializable. I can do so at every call to `np.clip`, but that (miniscule) inefficiency bugs me :bug: :laughing: